marketing/signals.py

```diff
@@ -0,0 +1,12 @@
+from django.db.models.signals import post_save
+from django.dispatch import recievers
+
+from registration.models import BetaTestApplication
+
+@recievers(post_save, sender=BetaTestApplication)
```

Comment:
Instead of adding the user on registration time, why not just add them once the first appserver is provisioned (and add a timestamp in the model)?

There's a method that does the same thing on the registration module: https://github.com/open-craft/opencraft/blob/b7090d2654a4228805819e5fb28631f451c481f1/registration/approval.py#L64

This will help make the filtering/sorting less complex on the `send_followup_emails` much easier. Since we're waiting the first appserver to add a user to the "mailing" list.